src/components/LinksList/index.js

```diff
@@ -18,22 +21,21 @@ class LinksList extends React.Component {
     } = this.props;
     return (
       <div>
-      { selectedMoc && selectedMoc.govtrack_id ? 
-        <Card 
-          className="actions-section" 
-          title="Actions:" 
-          extra={<div> <Button href="#" onClick={() => handleAutoFillMember(selectedMoc.govtrack_id)}>AutoFill Form</Button> <Button  onClick={() => this.props.showConfirm()}>No Events</Button></div>} 
-          style={{ width: 'auto' }}
-        /> : 
-        <div></div>
-      }
+        { selectedMoc && selectedMoc.govtrack_id && 
+        (<Card
+            className="actions-section"
+            title="Actions:"
+            extra={<div> <Button href="#" onClick={() => handleAutoFillMember(selectedMoc.govtrack_id)}>AutoFill Form</Button> <Button onClick={() => this.props.showConfirm()}>No Events</Button></div>}
+            style={{ width: 'auto' }}
+          />)
+        }
```

Comment:
@Alan-mag if you dont want to render anything you can just do a conditional this way and if it's false it will return null.